main.js

```diff
@@ -56,7 +56,38 @@ const VideoList = {
           source:'https://www.youtube.com/embed/MhkGQAoc7bc',
           author: 'Someone\'s Buddy'  
         }
-      ]
+      ],
+      navbarItems: [
+        {
+          name: 'Home',
+          src: 'https://www.realdevsquad.com',
+        },
+        {
+          name: 'Welcome',
+          src: 'https://welcome.realdevsquad.com',
+        },
+        {
+          name: 'Events',
+          src: 'https://www.realdevsquad.com/events.html',
+        },
+        {
+          name: 'Members',
+          src: 'https://members.realdevsquad.com',
+        },
+        {
+          name: 'Crypto',
+          src: 'https://crypto.realdevsquad.com',
+        },
+        {
+          name: 'Status',
+          src: 'https://status.realdevsquad.com',
+        },
+        {
+          name: 'Learn',
+          src: 'https://learn.realdevsquad.com',
```

Comment:
Thanks for addressing my comment
However, this may not be the required solution 

Essentially on "Learn" we want to redirect to the page it is currently on which can be achieve with a simple <a href="/"> in the html itself

Reply:
Fixed!

Reply:
Oops, I meant `href="#"`

Reply:
It's href="#" now.